src/components/ScorecardServiceRefLink/ScorecardServiceRefLink.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 Cortex Applications, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { useRouteRef } from '@backstage/core-plugin-api';
+import {scorecardServiceDetailsRouteRef} from '../../routes';
```

Comment:
do you have prettier set up? these should have spaces between the brackets added automatically.

![Screen Shot 2022-02-11 at 11 22 47 AM](https://user-images.githubusercontent.com/8764399/153628957-e48d0152-e541-483b-a072-8b07d114529a.png)

Reply:
I forgot to set up prettier, thanks for the heads-up.